README.md

```diff
@@ -1,24 +1,129 @@
-# README
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-Things you may want to cover:
+# 📖 Rails React Starter <a name="about-project"></a>
 
-* Ruby version
+It is a simple web application demonstrating use of React with Rails.
 
-* System dependencies
+## 🛠 Built With <a name="built-with"></a>
 
-* Configuration
+### Tech Stack <a name="tech-stack"></a>
 
-* Database creation
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>React</li>
+  </ul>
+</details>
 
-* Database initialization
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby-on-Rails</li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgres</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- Fetch and dispay greetings from API
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+Clone this project to your desired folder.
+
+```
+    git clone https://github.com/Progress2002/hello-rails-react.git
+```
+
+### Install
+
+Run from a terminal
+
+```
+    cd rails_react_starter
+    bundle install
+    yarn add
+```
+
+### Usage
+
+**To run the project follow the following instruction:-**
+
+```
+    rails db:setup
+    yarn build
+    rails s
+```
+
```

Comment:
- [ ] [OPTIONAL] I got an error after running `./bin/dev` but I was able to resolve this by running `rails assets:precompile` so I will recommend that you state the following commands in this order for end user:

``` ruby
yarn install

bundle install

rails db:drop

rails db:create

rails assets:precompile

rails db:migrate

./bin/dev
```

Reply:
Hello @Whoistolu, thanks for taking the time to review my project and providing code snippets for improvement